steely/plugins/haskell.py

```diff
@@ -0,0 +1,42 @@
+# External Dependencies:
+# - firejail (for sandboxing)
```

Comment:
Could you add this + GHC as dependencies to the project README or something?

Reply:
won't be needed if I do this 'disabled unless enabled' concept thing in the in project's config.

Reply:
It's still good to get the dependencies explicit somewhere. In some file in this particular dir? Or even just add an install script?

Reply:
Install script > repeating dependencies everywhere

Reply:
you mean something like `plugins/_haskell_install.sh`?

Reply:
This is probably the place and time where plugin directories become suitable.

Reply:
Do you want an install script?

Reply:
Would it need to be distro agnostic? If so I don't think it's worth it,
maybe just a _haskell_readme.md? Or folders for plugins

Reply:
The comment at the top of the haskell plugin clearly outlines the external dependencies. I don't think a new doc is going to help at all (and will add needless gunk)

Reply:
Correct. Disto-agnostic scripts are hard. At this point, I'm satisfied with the current approach:

List the dependencies.
Let the user install them manually.
Don't duplicate dependency comments everywhere.

Reply:
BTW: ghc isn't listed as a dependency here.

Reply:
Thanks. This is the problem with rewriting the dependencies everywhere. I wrote it on github and forgot to put it at the top of the script.

Reply:
You'd still only need to write them in one place...